Base/Common.xcconfig

```diff
@@ -68,6 +68,12 @@ CLANG_WARN_OBJC_ROOT_CLASS = YES_ERROR
 // Whether to warn on suspicious implicit conversions
 CLANG_WARN_SUSPICIOUS_IMPLICIT_CONVERSION = YES
 
+// Warn about incorrect uses of nullable values
+CLANG_WARN_NULLABLE_TO_NONNULL_CONVERSION = YES
+
+// Warn for missing nullability attributes
```

Comment:
I'm all for this attribute, but this might be seem as a "regression" by some if it produces too much noise.

I'm going to merge this just because I think it's the right setting to have moving forward. Users can disable it in their settings if its indeed too noisy.